sox.features.js

```diff
@@ -264,73 +264,63 @@
             $(document).on('sox-new-comment', colour);
         },
 
-        kbdAndBullets: function() {
-            // Description: For adding buttons to the markdown toolbar to surround selected test with KBD or convert selection into a markdown list
 
+        // Description: For adding buttons to the markdown toolbar to surround selected test with KBD or convert selection into a markdown list
+        kbdAndBullets: function() {
             function addBullets($node) {
                 var list = '- ' + $node.getSelection().text.split('\n').join('\n- ');
                 $node.replaceSelectedText(list);
+                StackExchange.MarkdownEditor.refreshAllPreviews();
```

Comment:
need to test if this will work on FF, because it didn't used to like us using `StackExchange` directly. If not, the `sox.Stack` object should do.